pkgs/applications/virtualization/qemu/user-static.nix

```
@@ -0,0 +1,51 @@
+{ lib, pkgsStatic }:
+
+let
+  qemuUserPlatforms = [
+    "aarch64-linux"
+    "armv7l-linux"
+    "i386-linux"
+    "powerpc-linux"
+    "powerpc64-linux"
+    "powerpc64le-linux"
+    "riscv32-linux"
+    "riscv64-linux"
+    "x86_64-linux"
+  ];
+
+  qemuTargets = map (system: (lib.systems.elaborate { inherit system; }).qemuArch + "-linux-user") qemuUserPlatforms;
```

```suggestion
  qemuTargets = map (system: (lib.systems.elaborate { inherit system; }).qemuArch + "-user") qemuUserPlatforms;
```

Nothing Linux-specific here, as QEMU also supports user-mode emulation of other (BSD) operating systems.

---

This was marked resolved, but nothing seems to have happened?

---

Oops sorry, I thought I took care of everything but I forgot about this.

---

I tried and it didn't work. The targets must have the `-linux-user` or `-bsd-user` suffixes. We can add `-bsd-user` after testing in a separate PR.

---

Oh yes I'm sorry, I misunderstood what this code did.  My bad.